src/index.html

```diff
@@ -1,15 +1,215 @@
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" onsubmit="onSubmit()">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <section class="box">
+          <label for="surname" class="form-field">
+            Surname:
+            <input type="text"
+              id="surname"
+              name="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="name" class="form-field">
+            Name:
+            <input type="text"
+              id="name"
+              name="name"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="age" class="form-field">
+            How old are You?
+            <input type="number"
+              id="age"
+              name="age"
+              value="12"
+              min="5"
+              max="100"
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="date" class="form-field">
+            Full date of birth:
+            <input type="date"
+              id="date"
+              name="date"
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="terms" class="form-field">
+            I accept the term of the agreement
+            <input type="checkbox"
+              id="terms"
+              name="terms"
+            >
+          </label>
+        </section>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <section class="box">
+          <label for="email" class="form-field">
+            E-mail:
+            <input type="email"
+              id="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="password"
+            class="form-field">
+            Password:
+            <input type="password"
+              id="password"
+              name="password"
+              min="8"
+              max="20"
+              required
+              autocomplete="off"
+              minlength="6"
+              maxlength="20"
+            >
```

Comment:
this input type doesn't have min & max attributes, only minlength & maxlength

```suggestion
            <input type="password"
              id="password"
              name="password"
              required
              autocomplete="off"
              minlength="6"
              maxlength="20"
            >
```